HiP-Achievements/Controllers/ActionBaseController.cs

```diff
@@ -20,17 +18,16 @@ namespace PaderbornUniversity.SILab.Hip.Achievements.Controllers
     public abstract class ActionBaseController<TArgs> : BaseController<TArgs> where TArgs : ActionArgs, new()
     {
         // ReSharper disable All
-        protected readonly EntityIndex _entityIndex;
-        protected readonly EventStoreService _eventStore;
+        protected readonly UserStoreService _userStoreService;
         // ReSharper Restore All
 
 
-        public ActionBaseController(EventStoreService eventStore, InMemoryCache cache)
+        public ActionBaseController(UserStoreService userStoreService)
```

Comment:
As not all actions are saved in the UserStore  the UserStoreService should only be used by the controllers that actually use it. Therefore the EventStoreService and EntityIndex should stay here